src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -345,8 +364,12 @@ private IRoleClaimStore<TRole> GetClaimStore()
             {
                 throw new ArgumentNullException("role");
             }
-            await claimStore.AddClaimAsync(role, claim, CancellationToken);
-            return await LogResultAsync(await UpdateRoleAsync(role), role);
+
+            using (Logger.BeginScope(await GetScopeMessageforRoleAsync(role)))
```

Comment:
Can we move this creation of the scope to to a private helper, should make life easier if we ever have to tweak the scope